### PR TITLE
feat(scss) Comm in app

### PR DESCRIPTION
**Pull request by AlexiaAlarza**
Creation of modal, toasts and banners of the new commIn App design.

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1240

File: packages/scss/demo/components/comm-in-app.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+<code class="code mod-block">&lt;div class="toasts-visual"&gt;
+	&lt;svg class="toasts-blob" xmlns="http://www.w3.org/2000/svg"&gt;
+		&lt;use href="../images/commInApp.svg#bannerBackground"&gt;&lt;/use&gt;
+	&lt;/svg&gt;
+	&lt;svg class="toasts-illustration" xmlns="http://www.w3.org/2000/svg"&gt;
+		&lt;use&gt; href="../images/commInApp.svg#comminapp-illu"&gt;&lt;/use&gt;
+	&lt;/svg&gt;
+&lt;/div&gt;	
+&lt;div class="toasts-content"&gt;
+    &lt;div class="toasts-content-title"&gt;&lt;strong&gt;Le module fait peau neuve !&lt;/strong&gt;&lt;/div&gt;
```
Comment (BertrandPodevin) [inline packages/scss/demo/components/comm-in-app.html:32]: english here

File: packages/scss/demo/components/comm-in-app.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+	<div class="communicationBanner">
+		<div class="communicationBanner-visual">
+			<svg class="communicationBanner-blob" xmlns="http://www.w3.org/2000/svg">
+				<use href="../images/commInApp.svg#bannerBackground"></use>
+			</svg>
+			<svg class="communicationBanner-illustration" xmlns="http://www.w3.org/2000/svg">
+				<use href="../images/commInApp.svg#comminapp-illu"></use>
+			</svg>
+		</div>
+		<span class="communicationBanner-title">Le module fait peau neuve, un nouvel affichage est disponible.</span>
```
Comment (BertrandPodevin) [inline packages/scss/demo/components/comm-in-app.html:53]: english here

File: packages/scss/demo/components/comm-in-app.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+		<div class="communicationBanner-visual">
+			<svg class="communicationBanner-blob" xmlns="http://www.w3.org/2000/svg">
+				<use href="../images/commInApp.svg#bannerBackground"></use>
+			</svg>
+			<svg class="communicationBanner-illustration" xmlns="http://www.w3.org/2000/svg">
+				<use href="../images/commInApp.svg#comminapp-illu"></use>
+			</svg>
+		</div>
+		<span class="communicationBanner-title">
+			Le module fait peau neuve, un nouvel affichage est disponible.
```
Comment (BertrandPodevin) [inline packages/scss/demo/components/comm-in-app.html:85]: english

File: packages/scss/src/components/_button.scss [hunk 4]
```diff
@@ -220,6 +220,11 @@
 			color: _color("primary");
 		}
 
+		&.mod-white {
+			background-color: white;
```
Comment (BertrandPodevin) [inline packages/scss/src/components/_button.scss:224]: _color("white");

File: packages/scss/src/components/_communicationBanner.scss [hunk 5]
```diff
@@ -0,0 +1,50 @@
+.communicationBanner{
+    background-color: _color('primary', '700');
+    color: white;
+    display: flex;
```
Comment (BertrandPodevin) [inline packages/scss/src/components/_communicationBanner.scss:4]: add a `flex-wrap: wrap;` here for smaller resolutions

File: packages/scss/src/components/_communicationBanner.scss [hunk 6]
```diff
@@ -0,0 +1,50 @@
+.communicationBanner{
+    background-color: _color('primary', '700');
+    color: white;
```
Comment (BertrandPodevin) [inline packages/scss/src/components/_communicationBanner.scss:3]: _color("white");

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1240

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1240

**Review by BertrandPodevin — APPROVED** [review]

**Review by lucienbertin — APPROVED** [review]

**Pull request merged**